server/src/main/java/org/elasticsearch/index/translog/TranslogStats.java

```diff
@@ -80,8 +80,12 @@ public void add(TranslogStats translogStats) {
         this.translogSizeInBytes += translogStats.translogSizeInBytes;
         this.uncommittedOperations += translogStats.uncommittedOperations;
         this.uncommittedSizeInBytes += translogStats.uncommittedSizeInBytes;
-        this.earliestLastModifiedAge =
-            Math.min(this.earliestLastModifiedAge, translogStats.earliestLastModifiedAge);
+        if (this.earliestLastModifiedAge < 1) {
```

Comment:
Let's use `this.earliestLastModifiedAge == 0`.

Reply:
Updated, thanks.